airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -13,22 +13,27 @@
 from airbyte_cdk.sources.streams.http import HttpStream, HttpSubStream
 from requests.exceptions import HTTPError
 
+DEFAULT_PAGE_SIZE = 100
```

Comment:
Why do you have different default value here and in `spec.json`?

Reply:
This is the page size for all streams. Previously, we specified it in the parent stream (100) and redefined it for the child streams (4 large streams with a page size of 30/50), I simplified it a bit, now everything is specified in the parent type.